docs/v2-system-addon/data_events.md

```diff
@@ -123,8 +135,11 @@ A user event ping includes some basic metadata (tab id, addon version, etc.) as
 ```js
 {
   "event": "BOOKMARK_ADD",
-  "source": "TOP_SITES",
+  "source": "HIGHLIGHTS",
   "action_position": 2,
+  "value": {
+    "card_type": ["history" | "trending"]
```

Comment:
nit: perhaps s/trending/pocket/ ?

Reply:
The current type of the cards from the Recommended by Pocket section is "trending". This is not the same as the "items you have in your pocket list" that just landed. Maybe I should update to include that as well.

Reply:
Sounds good, let's stick to "trending" then, leave the "pocket" for the future use (such as highlights).